src/vl-tabs.js

```diff
@@ -146,9 +153,28 @@ export class VlTabs extends vlElement(HTMLElement) {
     }
   }
 
+  _moveTabPaneTitleSlot(tabPane) {
+    const tabPaneTitleSlot = tabPane.querySelector(`[slot="${tabPane.id}-title-slot"]`);
+    if (tabPaneTitleSlot) {
+      const clonedTabPaneTitleSlot = tabPaneTitleSlot.cloneNode(true);
+
+      const existingTabPaneTitleSlot = this.querySelector(`vl-tabs > [slot="${tabPane.id}-title-slot"]`);
+      if (existingTabPaneTitleSlot) {
+        this.replaceChild(clonedTabPaneTitleSlot, existingTabPaneTitleSlot);
```

Comment:
Ik vermoed dat deze code nooit uitgevoerd wordt aangezien de selector nooit resultaat geeft. De code is bijgevolg overbodig?

Reply:
als ik de code debug kom ik hier weldegelijk in. Dit stuk is toegevoegd geweest om wijzigingen die in het tabsPane title slot gebeuren te laten doorkomen. Als we enkel adden komt bij elke wijziging er gewoon een nieuwe title bij. Door de replace vangen we dat op. Zie test/unit/vl-tabs.test.html